pkg/codegen/internal/test/testdata/resource-options.pp.py

```diff
@@ -0,0 +1,12 @@
+import pulumi
+import pulumi_aws as aws
+import pulumi_pulumi as pulumi
+
+provider = pulumi.providers.Aws("provider", region="us-west-2")
+bucket1 = aws.s3.Bucket("bucket1", opts=ResourceOptions(provider=provider,
+    dependsOn=[provider],
```

Comment:
in python the resource options are snake_cased, i.e. "depends_on" and "ignore_changes"

Reply:
Agh, good catch!

Reply:
Fixed.